src/components/ColorTokenTable/color-token-table.scss

```diff
@@ -0,0 +1,163 @@
+.color-token-table {
+  display: flex;
+  flex-direction: column;
+}
+
+.color-token-table__theme-switcher {
+  width: 100%;
+  height: rem(48px);
+  margin-bottom: $spacing-xl;
+  background: $ui-02;
+
+  // @include breakpoint('md') {
+  //   justify-content: flex-start;
+  //   background: transparent;
+  // }
+}
+
+.color-token-table__theme-switcher--sticky {
+  position: fixed;
+  top: 64px;
+  z-index: 9999;
+  max-width: 1164px;
+  box-shadow: 0px 2px 6px 0px rgba(#000000, 0.2);
+
```

Comment:
This part with the media queries is pretty gross, trying to think of a better way to do this